sormas-backend/src/main/java/de/symeda/sormas/backend/contact/Contact.java

```diff
@@ -1087,4 +1087,9 @@ public String getQuarantineChangeComment() {
 	public void setQuarantineChangeComment(String quarantineChangeComment) {
 		this.quarantineChangeComment = quarantineChangeComment;
 	}
+
+	@Transient
+	public boolean isArchived() {
```

Comment:
@MateStrysewske Is it correct that contacts can never be archived?

Reply:
Currently contacts can't be manually archived, correct. They're automatically archived when their cases are archived. We of course need to allow users to directly archive contacts, that's already planned.